resources/scss/ceres/views/Templates/AddressSelect/_address-select.scss

```diff
@@ -56,6 +56,18 @@
                 box-shadow: inset 0 0 0 .1rem $primary;
             }
         }
+
+        .primaryAddress {
+            position: absolute;
+            width: 100%;
+            top: 28px;
+            color: $primary;
+            font-size: 13px;
+
+            .btn {
+                width: 100% !important;
+            }
+        }
```

Comment:
1. please use Kebab case for class names.
2. position, width, text-color can be set with html classes instead of css. (position-absolute, w-100, text-primary)
3. Please open a pull request in the plentyShop LTS Modern Plugin to add these changes there as well and add them too the azure card.